testing/mocks/resourceresolver-mock/src/main/java/org/apache/sling/testing/resourceresolver/MockResourceResolverFactory.java

```diff
@@ -23,15 +23,20 @@
 import java.util.Map;
 import java.util.Stack;
 
+import org.apache.jackrabbit.JcrConstants;
 import org.apache.sling.api.resource.LoginException;
 import org.apache.sling.api.resource.ResourceResolver;
 import org.apache.sling.api.resource.ResourceResolverFactory;
 import org.osgi.service.event.EventAdmin;
 
+import com.google.common.collect.ImmutableMap;
+
 /**
  * Simple resource resolver factory
  */
 public class MockResourceResolverFactory implements ResourceResolverFactory {
+    
+    private static final String JCR_ROOT="jcr:root";
```

Comment:
Actually I think this should be [rep:root](http://jackrabbit.apache.org/api/2.1/org/apache/jackrabbit/spi/commons/name/NameConstants.html#REP_ROOT). jcr:root seems to be [a dummy name used in XML serialization](http://jackrabbit.apache.org/api/2.1/org/apache/jackrabbit/spi/commons/name/NameConstants.html#JCR_ROOT)